templates/core/terraform/airlock/airlock_processor.tf

```diff
@@ -6,17 +6,27 @@ locals {
   version = replace(replace(replace(data.local_file.airlock_processor_version.content, "__version__ = \"", ""), "\"", ""), "\n", "")
 }
 
-# re-using the web api app plan
-data "azurerm_app_service_plan" "core" {
-  name                = "plan-${var.tre_id}"
+data "azurerm_application_insights" "core" {
+  name                = "appi-${var.tre_id}"
```

Comment:
Is appi is a typo?
```suggestion
  name                = "api-${var.tre_id}"
```

Reply:
maybe, but I need the exact name in the 'data' object. so I call it the way it was defined